api_tester/lib/api_tests/models/store_transaction_api_test.dart

```diff
@@ -11,13 +11,15 @@ class _StoreTransactionApiTest {
     Map<String, dynamic> json = transaction.toJson();
   }
 
-  void _checkConstructor(String revenueCatIdentifier, String productIdentifier,
-      String purchaseDate) {
+  void _checkConstructor(String transactionIdentifier, String revenueCatIdentifier, 
+      String productIdentifier, String purchaseDate) {
     StoreTransaction transaction =
-        StoreTransaction(revenueCatIdentifier, productIdentifier, purchaseDate);
+        StoreTransaction(transactionIdentifier, revenueCatIdentifier, 
+            productIdentifier, purchaseDate);
```

Comment:
This is actually a breaking change, but devs shouldn't generally be using this, other than maybe in tests... Any thoughts on what to do here? @RevenueCat/coresdk. IMO, I think it would be ok to ship this and mention it in the changelog, but I recognize a few people could potentially be affected...

Reply:
Who knows if somebody is using it, so I'd avoid breaking it. Is it possible to keep the older version for backwards compatibility?

Reply:
+1, if we break even a single developer's app, even in debug, accidentally, it would be a failure on our side

Reply:
reminder: [At RevenueCat, a customer can’t hold the phone the wrong way](https://www.revenuecat.com/blog/company/why-were-customer-obsessed-and-you-should-be-too/)
even if they _shouldn't_ be using it, if they _can_, we shouldn't break things

Reply:
Good point! I will see if we can use custom constructors to keep backwards compatibility for this. Thanks!